sdp.go

```diff
@@ -435,3 +444,53 @@ func haveApplicationMediaSection(desc *sdp.SessionDescription) bool {
 
 	return false
 }
+
+func answerExtMaps(remoteExtMaps map[string]map[int]sdp.ExtMap, localMaps map[string][]sdp.ExtMap) map[string][]sdp.ExtMap {
```

Comment:
LGTM! Also make sure to handle IDs here, looks like right now you are just checking URLs?

Reply:
Exts are added directly from the remote side here so ID will be correct.